package.json

```diff
@@ -13,5 +13,8 @@
   ],
   "devDependencies": {
     "concurrently": "^8.2.2"
+  },
+  "dependencies": {
+    "expo": "^49.0.18"
```

Comment:
expo dependency already exists in frontend workspace, I'm not sure if we need it at the root `package.json` 🤔

Reply:
Okay you can remove that then if it's not needed.